token/services/db/sql/querybuilder.go

```diff
@@ -202,6 +200,25 @@ func in(args *[]interface{}, field string, searchFor []interface{}) (where strin
 	return fmt.Sprintf("(%s)", strings.Join(argnum, " OR "))
 }
 
+func whereTokenIDs(args *[]interface{}, ids []*token.ID) (where string) {
+	switch len(ids) {
+	case 0:
+		return ""
+	case 1:
```

Comment:
Do we need to handle this as a special case? These produce the same execution plan: `tx_id = ...` versus `tx_id IN (...)` for one value. (Do not know about `tx_id = ... AND idx = ...` versus `(tx_id, idx) = (..., ...)` though

Reply:
not sure, honestly. We can handle this in another PR